### ISSUE TITLE
[med] Keyboard Navigation: Email is not accessible using keyboard.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8983]
## Action Performed:
Using Chrome + Jaws, open URL staging.new.expensify.com
Navigate to Setting using Tab key and press Enter to select it.
Navigate to any workspace using Tab key and press using Enter to select it.
Navigate to Pay bills using Tab key and press Enter to activate it.
Navigate to the email present on the page using Tab key.
Verify the focus order and observe the issue.

## Expected Result:
Every interactive element should receive focus. 'applause.expensifail.com@expensify.cash' should be accessible using tab key.

## Actual Result:
'applause.expensifail.com@expensify.cash' does not receive keyboard focus using tab key. As a result, the keyboard-dependent users find difficulty in accessing the element.

## Workaround:
Yes

## Area issue was found in:
Workspace->Pay bills

## Failed WCAG checkpoints
2.1.1

## User impact:
The keyboard-dependent users find difficulty in accessing the applause.expensifail.com@expensify.cash.

## Suggested resolution:
Make sure 'applause.expensifail.com@expensify.cash' should be accessible using tab key and Ensure to remove the tabindex="-1" to make it accessible for the keyboard users. 

Reference:
https://webaim.org/techniques/keyboard/tabindex#:~:text=The%20tabindex%20attribute%20of%201,should%20be%20focusable%20or%20not.&text=tabindex%20values%20of%201%2B%20must%20be%20avoided.

## Platform:

- Web
- Desktop App

**Version Number:** v1.1.57-17
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/168290180-2d01d029-c5d2-46e9-814a-6de50b300146.mp4

**Comment by marcaaron** [comment]
@stitesExpensify should this go into an accessibility milestone or something? Unsure where it belongs.

**Comment by stitesExpensify** [comment]
Nowhere for now. Just let it sit. We're getting around to what to do with them :)
